addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/InsecureJsfViewStatePassiveScanRule.java

```diff
@@ -254,4 +241,23 @@ public int getCweId() {
     public int getWascId() {
         return 14; // WASC Id - Server Misconfiguration
     }
+
+    private AlertBuilder createAlert(String viewState) {
```

Comment:
Better keep it in the same place, less changes.

Reply:
Should be moved back up to ~L205